.github/workflows/Dependencies Security Scan.yml

```diff
@@ -7,17 +7,18 @@ name: Dependency Review Pull request
 
 on:
   pull_request:
-    branches: [main]
+    branches:
+      - main-6x
 
 env:
   HEAD_REF:  ${{ GITHUB.HEAD_REF }}
   REF_NAME:  ${{ GITHUB.REF_NAME }}
-  REPOSITORY_DISPATCH_BRANCH: ${{ github.event.client_payload.branch }}  
+  REPOSITORY_DISPATCH_BRANCH: ${{ github.event.client_payload.branch }}
 
 jobs:
   scan-actions:
     name: Scan Actions
-    uses: ed-fi-alliance-oss/ed-fi-actions/.github/workflows/repository-scanner.yml@latest
+    uses: ed-fi-alliance-oss/ed-fi-actions/.github/workflows/repository-scanner.yml@main
```

Comment:
We changed from using `latest` to `main` sometime last year, as it is easier to manage that way.